samples/nrf_desktop/src/hw_interface/motion_optical.c

```diff
@@ -436,6 +439,58 @@ static void update_cpi(u16_t cpi)
 	}
 }
 
+static void update_downshift_time(u8_t reg_addr, u32_t time)
+{
+	/* Set downshift time:
+	 * - Run downshift time (from Run to Rest1 mode)
+	 * - Rest 1 downshift time (from Rest1 to Rest2 mode)
+	 * - Rest 2 downshift time (from Rest2 to Rest3 mode)
+	 */
+	u32_t maxtime;
+	u32_t mintime;
+
+	if (reg_addr == OPTICAL_REG_RUN_DOWNSHIFT) {
```

Comment:
for this kind of flow control please use `switch/case`

Reply:
done